larana/ParticleIdentification/PhysdEdx.h

```diff
@@ -0,0 +1,60 @@
+////////////////////////////////////////////////////////////////////////
+//
+// A class for dE/dx functions for likelihood based particleID
+//
+// sungbino@fnal.gov
+//
+////////////////////////////////////////////////////////////////////////
+#ifndef PHYSDEDX_H
+#define PHYSDEDX_H
+
+#include "Math/VavilovAccurate.h"
+#include "TF1.h"
+#include <map>
+
+namespace pid {
+  class PhysdEdx {
+
+  public:
+    PhysdEdx();
```

Comment:
A default-constructed `PhysdEdx` object is not usable (other than to call `SetPdgCode` on it...and in that case, you might as well just create a `PhysdEdx` object with the correct PDG code).  Because of that, I suggest removing the default constructor.

Reply:
Done.